UploadDaemon_Test/TraceFileScannerTest.cs

```diff
@@ -5,7 +5,7 @@
 using System.IO.Abstractions;
 using System.IO.Abstractions.TestingHelpers;
 using System.Linq;
-using UploadDaemon;
+using UploadDaemon.Scanning;
 
 [TestFixture]
 public class TraceFileScannerTest
```

Comment:
Move to `UplaodDaemon.Scanning`? If you do, remove corresponding using statement.